Client/Frontend/ContentBlocker/ContentBlockerHelper.swift

```diff
@@ -5,23 +5,118 @@
 import WebKit
 import Shared
 import Deferred
+import Data
+import BraveShared
 
 private let log = Logger.browserLogger
 
-enum BlocklistName: String {
-    case advertising = "disconnect-advertising"
-    case analytics = "disconnect-analytics"
-    case content = "disconnect-content"
-    case social = "disconnect-social"
-
-    var filename: String { return self.rawValue }
+// Rename to BlockList
+class BlocklistName: Equatable, Hashable {
+    static let ad = BlocklistName(filename: "block-ads")
+    static let tracker = BlocklistName(filename: "block-trackers")
+    static let https = BlocklistName(filename: "upgrade-http")
+    static let script = BlocklistName(filename: "block-scripts")
+    static let image = BlocklistName(filename: "block-images")
```

Comment:
I thought we weren't supporting block images in 1.7?

Reply:
We aren't, it will be disabled in the UI, but figured I'd just throw in the scripts since I was already modifying all of these rulesets. Block image script does not properly deal with all images, so we will need to combine this with injected scripts as well.